content/Platform/members.mdx

```diff
@@ -0,0 +1,35 @@
+---
+id: members
+slug: members
+title: Members
+---
+
+## Adding Users
+
+Users can be added to Configu using the following methods:
+
+- Send Invite
+- SAML Single Sign-On
+
+New members are assigned the [default role](../authorization#default-role) from the organization settings.
+
+## Send Invite
+
+Users can be manually invited to join an organization from the member's page. The account must be able to receive mail.
+
+![image](./img/members/members-invite.png)
+
+## SAML Single Sign-On
+
+Configu supports SAML Single Sign-On for managing organization access and authentication using your Single Sign-On provider.
+For more information about SAML support please contact us at info@configu.com.
+
+## Assign roles to a Member
+
+Organization members can be given roles that define what resources they can access. A member is always assigned a predefined role.
+![image](./img/members/members-edit-roles.png)
+![image](./img/members/members-edit-roles-alert.png)
```

Comment:
Please add an explanation of the multiple role combinations and of the relationship between selecting a predefined role (which I must select) and adding a custom role as well

Reply:
I'm thinking of rewriting the "Assign roles" section on the authorization page to have these details including an example because it's true for both members and tokens. Here is what I'm thinking of writing there:

```
## Assign roles

Members and tokens always have a predefined role that acts as their base permissions. Custom roles give stacking permissions on top of the predefined role permissions. For example:

- User "John Doe" has the "Reader" predefined role and a custom role named "Developer" that has the read permission for the "dev" config set.
- "John Doe" can:
  - Read config values of all config sets due to the "Reader" role.
  - Write config values to the "dev" config set and all its children due to the "Developer" Role

To assign roles to your [organization members](../members#assign-roles-to-a-member) or [organization tokens](../tokens#assign-roles-to-a-token), go to their respective pages and edit their roles.
```

Do you think it will make this clear?